dts/arm/raspberrypi/rpi_pico/rp2350.dtsi

```diff
@@ -0,0 +1,402 @@
+/*
+ * Copyright (c) 2024 Andrew Featherstone
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <arm/armv8-m.dtsi>
+#include <zephyr/dt-bindings/adc/adc.h>
+#include <zephyr/dt-bindings/gpio/gpio.h>
+#include <zephyr/dt-bindings/clock/rpi_pico_rp2350_clock.h>
+#include <zephyr/dt-bindings/i2c/i2c.h>
+#include <zephyr/dt-bindings/regulator/rpi_pico.h>
+#include <zephyr/dt-bindings/reset/rp2350_reset.h>
+#include <mem.h>
+
+#ifndef RPI_PICO_DEFAULT_IRQ_PRIORITY
+#define RPI_PICO_DEFAULT_IRQ_PRIORITY 7
+#endif
```

Comment:
should this be a Kconfig option?

Reply:
Same as https://github.com/zephyrproject-rtos/zephyr/pull/77368#discussion_r1793158385.
So, I mark it as resolved.